src/_capi.pxd

```diff
@@ -568,22 +597,40 @@ cdef extern from "blend2d.h":
+#    BLResult blImageConvert(BLImageCore* self, uint32_t format)
+#    bool blImageEquals(const BLImageCore* a, const BLImageCore* b)
+#    BLResult blImageScale(BLImageCore* dst, const BLImageCore* src, const BLSizeI* size, uint32_t filter, const BLImageScaleFilter* options)
+
+
     # BLImage
-    BLResult blImageInit(BLImageCore* self)
-    BLResult blImageInitAs(BLImageCore* self, int w, int h, uint32_t format)
-    BLResult blImageInitAsFromData(BLImageCore* self, int w, int h, uint32_t format, void* pixelData, intptr_t stride, BLDestroyImplFunc destroyFunc, void* destroyData)
-    BLResult blImageDestroy(BLImageCore* self)
-    BLResult blImageReset(BLImageCore* self)
-    BLResult blImageAssignMove(BLImageCore* self, BLImageCore* other)
-    BLResult blImageAssignWeak(BLImageCore* self, const BLImageCore* other)
-    BLResult blImageAssignDeep(BLImageCore* self, const BLImageCore* other)
-    BLResult blImageCreate(BLImageCore* self, int w, int h, uint32_t format)
-    BLResult blImageCreateFromData(BLImageCore* self, int w, int h, uint32_t format, void* pixelData, intptr_t stride, BLDestroyImplFunc destroyFunc, void* destroyData)
-    BLResult blImageGetData(const BLImageCore* self, BLImageData* dataOut)
-    BLResult blImageMakeMutable(BLImageCore* self, BLImageData* dataOut)
-    BLResult blImageConvert(BLImageCore* self, uint32_t format)
-    bool blImageEquals(const BLImageCore* a, const BLImageCore* b)
-    BLResult blImageScale(BLImageCore* dst, const BLImageCore* src, const BLSizeI* size, uint32_t filter, const BLImageScaleOptions* options)
+    BLResult blImageInit(BLImageCore* self) ;
+    BLResult blImageInitAs(BLImageCore* self, int w, int h, BLFormat format) ;
+    BLResult blImageInitAsFromData(BLImageCore* self, int w, int h, BLFormat format, void* pixelData, intptr_t stride, BLDataAccessFlags accessFlags, BLDestroyExternalDataFunc destroyFunc, void* userData) ;
+    BLResult blImageDestroy(BLImageCore* self) ;
+    BLResult blImageReset(BLImageCore* self) ;
+    BLResult blImageAssignMove(BLImageCore* self, BLImageCore* other) ;
+    BLResult blImageAssignWeak(BLImageCore* self, const BLImageCore* other) ;
+    BLResult blImageAssignDeep(BLImageCore* self, const BLImageCore* other) ;
+    BLResult blImageCreate(BLImageCore* self, int w, int h, BLFormat format) ;
+    BLResult blImageCreateFromData(BLImageCore* self, int w, int h, BLFormat format, void* pixelData, intptr_t stride, BLDataAccessFlags accessFlags, BLDestroyExternalDataFunc destroyFunc, void* userData) ;
+    BLResult blImageGetData(const BLImageCore* self, BLImageData* dataOut) ;
+    BLResult blImageMakeMutable(BLImageCore* self, BLImageData* dataOut) ;
+    BLResult blImageConvert(BLImageCore* self, BLFormat format) ;
+    bool blImageEquals(const BLImageCore* a, const BLImageCore* b) ;
+    BLResult blImageScale(BLImageCore* dst, const BLImageCore* src, const BLSizeI* size, BLImageScaleFilter filter);
```

Comment:
Same thing here. I prefer not to have large chunks of this file commented out, more so when there are similar versions of the lines which are not commented out.